src/otx/core/data/module.py

```diff
@@ -103,6 +105,17 @@ def _get_dataset(self, subset: str) -> OTXDataset:
             raise KeyError(msg)
         return dataset
 
+    def _postprocess(self, dataset: Dataset) -> Dataset:
+        """Post processing after importing dataset from Datumaro."""
+        if self.task == "DETECTION":
+            invalid_ids = [(item.id, item.subset) for item in dataset.get_subset("train") if len(item.annotations) == 0]
+            # TODO(Jaeguk): Add data filtering for invalid bbox  # noqa: TD003
+            # TODO(Jaeguk): Delete unused labels  # noqa: TD003
+            for invalid_id in invalid_ids:
+                dataset.remove(invalid_id[0], invalid_id[1])
+
+        return dataset
+
```

Comment:
Would you please move this function to a separate file such as `pre_filtering.py`. I'm feeling that this can be scaled according to the task type. In addition, since we upgraded to Datumaro 1.6.0rc1, consider to use [filtering by a user-provided Python function query](https://openvinotoolkit.github.io/datumaro/latest/docs/level-up/intermediate_skills/09_data_filtering.html).
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/8fb89e8e-af1d-4fc0-87cf-5f9d28152ebd)

Reply:
Done by [6181efd](https://github.com/openvinotoolkit/training_extensions/pull/2823/commits/6181efd111b4b8eecd781ab9cf78eda2fc40d87c)